client/src/components/event/filter-bar.js

```diff
@@ -50,8 +47,8 @@ class DropdownGroup extends React.Component {
   }
```

Comment:
Shared button?

Reply:
According to new task description you don't need to do refactoring, so my previous comment isn't relevant